packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_service_runs.py

```diff
@@ -180,4 +180,17 @@ class ResourceTrackerServiceRunStatus(str, enum.Enum):
         nullable=False,
         doc="Timestamp when was the last heartbeat",
     ),
+    sa.Column(
+        "service_run_status_msg",
+        sa.String,
+        nullable=True,
+        doc="Custom message/comment, for example to help understand root cause of the error during investigation",
```

Comment:
question, this is an error of the resource tracking?

Reply:
Yes, but doesn't need to be, my idea was that in the future for example director-v2/sidecar might tell RUT what kind of error occurred.